lyluatex.lua

```diff
@@ -497,8 +496,7 @@ function Score:header()
         )
     elseif self.insert == 'systems' then
 	header = header:gsub(
-	    [[<<<PREAMBLE>>>]], [[\include "lilypond-book-preamble.ly"]]):gsub(
-	    [[<<<PAPER>>>]], [[indent = 0\mm]])
```

Comment:
I don't understand this change: in `lilypond-book`, `\noindent` is the default, but it doesn't have a `fullpage` mode. So IMHO, it would be a good thing to have `indent = 0\mm` for `\lily` and `ly`, while auto indent is a good choice for bigger scores included by `\lilypondfile`.